internal/hud/webview/convert.go

```diff
@@ -143,6 +121,11 @@ func tiltfileResourceView(s store.EngineState) Resource {
 }
 
 func resourceInfoView(mt *store.ManifestTarget) ResourceInfoView {
+	if mt.Manifest.Name == model.UnresourcedYAMLManifestName {
```

Comment:
We've made this check a couple of times. Should we make it a method on the manifest?

Reply:
ooh maybe!